### DISCUSSION TITLE
List menu with icons

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#471] [Q&A] [answered]
We can add icons to the list menus for better and faster recognition of the available actions.

<img src="https://github.com/devemux86/cruiser/assets/3484020/fd02caf8-0fe3-4a48-be37-593c3f03bb4b" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/64581eb8-4119-4a47-9c25-5d3bb73fba86" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/dfa08d63-9054-4bda-bdbb-2da5931befb6" width="33%"/>

**Comment by umpfmpf** [comment]
It looks a bit clearer and more self-explanatory.

**Comment by airald** [comment]
will be useful, sometimes is more complicated than expected just read text, the icons are brain friendy.

**Comment by devemux86** [comment, marked as answer]
List menu with icons is available in [Cruiser 3.0.42](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
